src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1238,10 +1238,11 @@ impl Coordinator {
                                 // problem, so we don't want a notice.
                                 !ids_set.contains(&ObjectId::Item(**dependant_id))
                             })
-                            .map(|dependant_id| {
-                                humanizer
-                                    .humanize_id(*dependant_id)
-                                    .unwrap_or(id.to_string())
+                            .flat_map(|dependant_id| {
+                                // If we are not able to find a name for this ID it probably means
+                                // we have already dropped the compute collection, in which case we
+                                // can ignore it.
+                                humanizer.humanize_id(*dependant_id)
```

Comment:
I made this change because collections in the controller are now cleaned up less frequently, so it's more likely to find ones that were already dropped (it was possible before I think). This made some test fail because we would print a notice like "this index is still used by my_other_index and u2".